terraform/modules/static-frontend/cdn.tf

```diff
@@ -0,0 +1,130 @@
+# Access identity, policy and document for the CDN
+
+locals {
+  media_s3_origin_id = "S3-${var.project}-${var.environment}-assets-${local.unique_string}"
+}
+
+resource "aws_cloudfront_origin_access_identity" "origin_access_identity" {
```

Comment:
this is the origin access identity for the cloudfront cdn. it allows the cdn to access objects in the bucket